js/scenes/Surveys/SurveysContainer.js

```diff
@@ -1,13 +1,184 @@
 import React, { Component } from 'react';
-import Surveys from './Surveys';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Dimensions, Image } from 'react-native';
+
+import '../../redux/modules/actions/questionsActions';
+import Surveys from './Pages/Surveys';
+import CommentScreen from './Pages/CommentScreen';
+import Loader from '../../components/Loader';
+import {
+  setSurveyAnswers,
+  setSurveyComments,
+  setTalkData
+} from '../../redux/modules/actions/surveyActions';
+import { goToSurveyComplete } from '../../navigation/navHelpers';
+import { giveFeedback } from '../../config/surveyhelpers';
+import { parseGoalsObjToArr } from '../Lib/helperFunctions';
 
 class SurveysContainer extends Component {
+  static route = {
+    navigationBar: {
+      title: <Image
+        style={{ width: Dimensions.get('window').width / 2.5, height: 27 }}
+        source={require('../../assets/images/beta_talks_logo_white_header.png')}
+      />
+    }
+  }
+
+  constructor(props) {
+    super(props);
```

Comment:
you can remove the `constructor` all together.

Reply:
Done